src/03-dice-roller/DiceRoller/DiceRoller/Program.cs

```diff
@@ -10,35 +10,37 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("> /roll 3d6 2d8");
+            // ask user input and make string array
+            Console.WriteLine("Insert command and dices to roll.");
             Console.WriteLine();
+            Console.Write(">");
+            string[] inputArray = Console.ReadLine().Split(null);
 
-            DiceRoller diceRoller = new DiceRoller();//diceroller object
+            //pass this array to parser method and return list of dices
+            DiceRollCommandParser parser = new DiceRollCommandParser();
+            List<Dice> dices = parser.ParseDices(inputArray);
 
-            // make new list of dices. Dices are pre-defined with sides and description.
-            List<Dice> dices = new List<Dice> { Dice.D6, Dice.D6, Dice.D6, Dice.D8, Dice.D8 };
+            //make list of dices - manual input
+            //List<Dice> dices = new List<Dice> {Dice.D6,Dice.D6,Dice.D6,Dice.D8,Dice.D8 };
 
-            //1) Make new diceRolls list to save results. 
-            //2) Roll the dice objects. Each dice has a parameter (sides, description).
-            List<DiceRoll> diceRolls = diceRoller.Roll(dices);//Roll will return the List of results
+            //diceroller object
+            DiceRoller diceRoller = new DiceRoller();
 
-            foreach (var diceRoll in diceRolls)//loop through the list of diceroll results
+            //roll the dices to get results
+            List<DiceRoll> results = diceRoller.Roll(dices);
+
+            //echo results
+            foreach (var r in results)
             {
-                //for each dice, show the result
-                Console.WriteLine($"1{diceRoll.DiceDesc}: {diceRoll.Value}");
+                Console.WriteLine($"1{ r.Description}, { r.Value}");
```

Comment:
I personally would say that `result.Description` is much more descriptive than `r.Description`.. If i'm only looking at r.Description I have to change my focus and look for a place where r is defined to get the context.. Why not abc.Description?